chingu-voyage4/Geckos-Team-32
# Database

## Purpose
*connected MongoDB and mLabs to App. Started  Route for user Sign up/Login with @Marylicious

## Approach/Fixes
* It gives us a way to store playlists, user accounts, themes, and popular music videos based on geolocation (if we get that far this voyage)

## Testing
* `npm Install`.
* Make .env file as previously shared and make sure it has the correct MongoDB URI credential. 
* Run `mongod`in your terminal before `npm Start`
* Make sure everything still works as it should
